services/motion/builtin/move_request.go

```diff
@@ -189,6 +225,98 @@ func (mr *moveRequest) deviatedFromPlan(ctx context.Context, waypoints state.Way
 	return state.ExecuteResponse{}, nil
 }
 
+// getTransientDetections returns a list of geometries in their relative position (with respect to the base)
+// and another list of geometries in their absolute positions (with respect to the world) as observed by camName.
+// Relative position geometries are used for constructing a plan, since the base plans in its own frame.
+// Absolute position geometries are used for checking a plan for collisions.
+func (mr *moveRequest) getTransientDetections(
+	ctx context.Context,
+	visSrvc vision.Service,
+	camName resource.Name,
+	currentPosition *referenceframe.PoseInFrame,
+) ([]*referenceframe.GeometriesInFrame, []*referenceframe.GeometriesInFrame, error) {
```

Comment:
Generally not a fan of returning multiple types here.  Is there an issue with returning only one of these and having caller of this function put it into the frame it needs?

Reply:
I do not think there is an issue with having the caller of the function transform the output. I will update this.